Sources/App/Models/EventReg/EventReg.generated.swift

```diff
@@ -33,8 +33,8 @@ extension EventReg {
     init(_ string: String) {
       switch string {
       case "approved": self = .approved
-      case "waiting": self = .waiting
       case "rejected": self = .rejected
+      case "waiting": self = .waiting
       default: self = .canceled
       }
     }
```

Comment:
I don't know, since it is generated, maybe we can leave it as is, but here we can use `enum: String` and make it much cleaner.

Reply:
hmm, probably yes... but for now lets leave it as is